apps/profilers/src/bin/block-serde-profiler.rs

```diff
@@ -12,8 +12,8 @@ const BLOCK_BYTES: &[u8] = include_bytes!("../../../../protocol/test-fixtures/sr
 fn main() -> anyhow::Result<()> {
     let matches = App::new("block-serde-app")
         .arg(
-            Arg::with_name("mode")
-                .short("m")
+            Arg::new("mode")
```

Comment:
Why not use with_name() and use new () twice? [Link](https://docs.rs/clap/2.20.5/clap/struct.Arg.html)